firmware/Core/Inc/telecommands/telecommand_args_helpers.h

```diff
@@ -7,4 +7,11 @@ uint8_t TCMD_ascii_to_uint64(const char *str, uint32_t str_len, uint64_t *result
 uint8_t TCMD_extract_uint64_arg(const char *str, uint32_t str_len, uint8_t arg_index, uint64_t *result);
 uint8_t TCMD_extract_string_arg(const char *str, uint8_t arg_index, char *result, uint16_t result_max_len);
 uint8_t TCMD_extract_hex_array_arg(const char *args_str, uint8_t arg_index, uint8_t result_array[], uint16_t result_array_size, uint16_t *result_length);
+
+uint8_t TCMD_ascii_to_double(const char *str, uint32_t str_len, double *result);
```

Comment:
Awesome work overall! Just a tiny issue here with duplicate function prototypes. 

Also, this would be a good opportunity for you to add your name to the list of contributors in the README :)